internal/cli/trigger_inspect.go

```diff
@@ -115,6 +117,11 @@ func (c *TriggerInspectCommand) Run(args []string) int {
 		tags = strings.Join(trigger.Tags[:], ", ")
 	}
 
+	var lastActiveTime string
```

Comment:
I like this idea! I think I'll go with `"n/a"` instead of unknown. Just because if the time hasn't been set, we know it hasn't been active. Where defaulting to unknown makes it seem like we can't know its active time.